business_register/converter/declaration.py

```diff
@@ -76,6 +77,24 @@ def __init__(self):
     def log_error(self, message):
         logger.error(f'Declaration id {self.current_declaration.nacp_declaration_id} : {message}')
 
+    def to_float(self, value):
+        if value not in self.NO_DATA:
+            return float(value)
+        return
+
+    def find_value(self, dictionary: dict, set_of_keys: set, default_value):
```

Comment:
I think `tuple` will be better than `set` here, tuple is lighter
```suggestion
    def find_value(self, dictionary: dict, default_value, keys: tuple[str]):
```

Reply:
I`ve chosen the set because of the uniqueness its elements)